### PR TITLE
`VaButton` fixes

### PR DESCRIPTION
**Pull request by aluarius**
## Description
- [x] Fixed bugs with `VaButtons` after merging dark theme functionality.
- [x] Fixed minor bug for `VaToggle` (different buttons height for icon-only and not icon-only states in one toggle group block).

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

**Comment by rustem-nasyrov** [comment]
Facing issues with colors in the sources.
image

**Comment by rustem-nasyrov** [comment]
Blue buttons have a black text. 
image

**Comment by rustem-nasyrov** [comment]
`Open in GitHub` button lead to the 404 page.
image

**Review by rustem-nasyrov — CHANGES_REQUESTED** [review]
Needs to be compared with a current version on vuestic.dev and fix the differences.

**Comment by m0ksem** [comment]
> `Open in GitHub` button lead to the 404 page. image

It is okay, since it is not on GitHub yet, but in 1.5.0 branch :)

File: packages/ui/src/components/va-button/VaButton.vue [hunk 1]
```diff
@@ -170,7 +170,7 @@ export default defineComponent({
     }))
 
     // styles
-    const isLowContrastBg = computed(() => props.plain || isLightBackground(colorComputed.value, props.backgroundOpacity))
+    const isLowContrastBg = computed(() => props.plain || isLightBackground(colorComputed.value, props.backgroundOpacity, 200))
```
Comment (m0ksem) [inline packages/ui/src/components/va-button/VaButton.vue:173]:
```suggestion
    const isTransparentBg = computed(() => props.plain || props.backgroundOpacity < 0.5)
```

File: packages/ui/src/services/color-config/color-functions.ts [hunk 2]
```diff
@@ -6,10 +6,13 @@ export const colorToRgba = (color: ColorInput, opacity: number) => {
   return new ColorTranslator(color).setA(opacity).RGBA
 }
 
-export const isLightBackground = (color: ColorInput, opacity = 1) => {
-  // TODO: replace with color mixin (doesn't play any role in dark theme)
+export const isLightBackground = (color: ColorInput, opacity = 0.6, edge = 120) => {
```
Comment (m0ksem) [inline packages/ui/src/services/color-config/color-functions.ts:9]: Do not use opacity here at all. Better use applyColors(actuallBg, bg)

Opacity 0.6 doesn't mean that is it light bg...

File: packages/docs/src/components/header/Header.vue [hunk 3]
```diff
@@ -127,48 +127,18 @@ export default class Header extends mixins(PropsMixin) {
...
-          justify-content: center;
-        }
-      }
     }
 
-    &__links {
+    &__prefences {
       display: flex;
+      align-items: center;
+      font-weight: $font-weight-bold;
```
Comment (m0ksem) [inline packages/docs/src/components/header/Header.vue:135]: Do not scss variables. Use css variables or hardocde :)

**Review by m0ksem — CHANGES_REQUESTED** [review]

**Comment by m0ksem** [comment]
> Blue buttons have a black text. image

I'll handle it.

**Comment by aluarius** [comment]
> Facing issues with colors in the sources. 

Not related to this pr, will move to different issue.

**Comment by aluarius** [comment]
> Needs to be compared with a current version on vuestic.dev and fix the differences.

It's impossible because current version contains old buttons implementation.

File: packages/ui/src/components/va-button/VaButton.vue (hunk 1 above)
Comment (aluarius) [reply packages/ui/src/components/va-button/VaButton.vue:173]: I planned to solve this while reimplementing buttons to after-before. But ok, let's make it here :)

File: packages/docs/src/components/header/Header.vue (hunk 3 above)
Comment (aluarius) [reply packages/docs/src/components/header/Header.vue:135]: It was here before - will remove.

**Review by m0ksem — APPROVED** [review]

**Pull request merged**